pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1961,18 +1961,21 @@ def _meth_InvokeMethod(self, methodname, objectname, Params, **params):
                 CIM_ERR_FAILED,
                 _format("Invalid Response. Must be list or tuple)",
                         type(result)))
-        for param in result[1]:
-            if not isinstance(param, CIMParameter):
+
+        # Map output params to NocaseDict to be compatible with return
+        # from _methodcall. The input list is an iterable of CIMParameters
+        output_params = NocaseDict()
+        rtn_params = result[1]
```

Comment:
First time I was reading this, I was confused about return vs output parameters, and that was caused by the variable name rtn_params. After a closer look, I agree with the code, but suggest to use a different name for that variable, eg out_param_list.

Reply:
Thats better because I spent an hour trying to figure out what was wrong.

Reply:
Made changes.   Also found some doc issues in MethodProvider when reviewing yesterday and fixed those.  Committed new version with all those changes in separate commit.